cli/compose/convert/service.go

```diff
@@ -158,6 +158,7 @@ func Service(
 			Placement: &swarm.Placement{
 				Constraints: service.Deploy.Placement.Constraints,
 				Preferences: getPlacementPreference(service.Deploy.Placement.Preferences),
+				MaxReplicas: service.Deploy.Placement.MaxReplicas,
```

Comment:
@vdemeester @thaJeztah any ideas why this one does not get value? Compose version 3.8 already allows me set **max_replicas_per_node** and if I hardcode value here ( ```MaxReplicas: 1```) it works correctly.

Reply:
@silvin-lubecki any ideas?

Reply:
ping @silvin-lubecki ?

Reply:
I honestly don't know. Can you provide a unit test showing this issue?

Reply:
Good point. It pointed out that this one needed *mapstructure* setting.